sass/_themes.scss

```diff
@@ -420,7 +421,8 @@ $themes: (
         HighestLowestMarkerLabelBg: $COLOR_DARK_GRAY_2,
         HighestLowestMarkerLabelText: $COLOR_WHITE,
         HighestSpotColor: $COLOR_GREEN_1,
-        LowestSpotColor: $COLOR_RED
+        LowestSpotColor: $COLOR_RED,
+        ChartBackgroundOpacity: 0.4
```

Comment:
The `ChartBackgroundOpacity` for white theme is 0.5, is this suppose to be 0.4?

Reply:
yes, it's correct to make it more visible in white theme. Anyway this background should be changed after the new one is done by the UI/UX team.